pkgs/by-name/ue/uesave/package.nix

```diff
@@ -0,0 +1,25 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "uesave";
+  version = "0.5.0";
+  src = fetchFromGitHub {
+    owner = "trumank";
+    repo = "uesave-rs";
+    rev = "v${version}";
+    hash = "sha256-9gOOSLejVfR1KJMhcNuKDkuTOvPC6sNG8xQOZlt8NxI=";
+  };
+
+  cargoHash = "sha256-U6RzSS2j6FK70OHlmWmHZZYT3UB0+Hi+uLofLy+XtGQ=";
+
```

Comment:
Also would be good to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook